packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -126,9 +128,8 @@
 }
 
 .main-content {
-  @apply grid grid-cols-[calc(100vw_-_1.5rem)] grid-rows-[min-content] gap-x-4 mt-16 min-h-[calc(100vh_-_4rem)];
-  @apply tablet:grid-cols-[minmax(min-content,500px)_minmax(20rem,1fr)] tablet:gap-x-8 tablet:mt-[80px] tablet:mb-auto tablet:mx-3;
-  @apply desktop:grid-cols-[600px_1fr];
+  @apply grid mt-16 mx-6;
```

Comment:
I don't believe mobile needed all the calculations and gap definitions since the `mx-6` should take care of that and components within `main-content` already have margin-top/bottom applied to them.